Automattic/wp-calypso
# Odie Client: add Zendesk conversation to chat

## Proposed Changes

* Add Smooch hook to Odie
* Add support provider state to manage chat between Odie and Zendesk
* Add feature flag to hide behind

## Why are these changes being made?

We are unifying the support chat experience. This involves merging the AI conversation (odie) and the human conversation (zendesk) into a singular chat window (odie-client). This is a very raw version of the chat experience. There are some known bugs and upgrades that need to be made. We are deploying in this state to allow others to work asynchronously.

## Testing Instructions

The main thing to test is for regression. All of the changes should exist behind the feature flag, but we need to make sure that the experience without the flag is the exact same as production. We should also be sure there are no additional requests to Zendesk.

To test the new chat experience add `?flags=help-center-experience` to the URL. It would be easiest using the Live Link probably.
Then start a conversation with odie and ask to talk to a human.
Click the Get Support button and you should get a new conversation within the Help Center.
